edx_rest_framework_extensions/tests/test_utils.py

```diff
@@ -38,10 +71,81 @@ def setUp(self):
         self.jwt = generate_jwt_token(self.payload)
 
     def test_decode_success(self):
-        self.assertDictEqual(jwt_decode_handler(self.jwt), self.payload)
+        """
+        Confirms that the format of the valid response from the token decoder matches the payload
+        """
+        self.assertDictEqual(utils.jwt_decode_handler(self.jwt), self.payload)
+
+    @ddt.data(*JWT_ISSUERS)
+    @override_settings(JWT_AUTH=JWT_AUTH_VALID_ISSUERS)
```

Comment:
Since you're overriding this for every test, update the actual settings in `test_settings.py`.